src/workerd/jsg/BUILD.bazel

```diff
@@ -86,6 +83,7 @@ wd_cc_library(
         "jsg.c++",
         "jsvalue.c++",
         "modules.c++",
+        "modules-new.c++",
```

Comment:
Can't we make "modules-new" a new target?

Reply:
generally no. It is needed in all the same places `modules.c++|h` are needed. I will look at cleaning this part up later (both for the original modules impl and the new one) in a separate follow up PR (there will be several PRs before this work is complete)